### ISSUE TITLE
Collection runner locks up when hitting and exiting from a request with no URL

### ISSUE THREAD

**Issue by Heydiddly** [a85/POSTMan-Chrome-Extension#3288]
Found a bit of a bug... I ran the collection runner and it hit a request I'd forgotten to remove which doesn't actually have a URL... would expect this to just be skipped but instead the runner got stuck :confused: fine, I go and delete the offending request and hit stop to run it again... and now the runner is stuck "stopping"!

**Comment by sdnts** [comment]
@Heydiddly I was able to reproduce this, I'll let you know here when a fix is out

**Comment by sivcan** [comment]
@Heydiddly I tried to reproduce this issue on the latest version of the app (v6.1.4) but the runner didn't get stuck for me.

Here's what I tried:
1. Ran an empty request
2. Put an empty request between the multiple requests and that still stops the runner at empty request (intended behaviour to stop the run) but the runner didn't get stuck.
3. Put an empty request between a lot of requests and started the run, meanwhile deleted the empty request from the collection sidebar in requester and came back to runner and stopped the run.
It stopped properly and the runner UI didn't get stuck.

If this issue still persists for you then let us know on the latest version of the postman app.
Thanks.
